css/pages/discord.css

```diff
@@ -1,39 +1,81 @@
-.main-heading{
+.discord-wrapper {
+  margin-left: 5%;
+}
+
+.main-heading {
+
   text-align: left;
   font-weight: 600;
-  padding: 10px;
+
+  margin: 40px 0 30px 0;
 }
-body{
-    margin-left: 5%;
+
+.get-started {
+  text-align: left;
+  font-weight: 550;
+  
+}
+.discord > *{
+  text-align: left;
+  font-weight: 500;
+  /* margin-left: 2%; */
```

Comment:
please remove commented code

Reply:
Removed the commented code https://github.com/Real-Dev-Squad/website-welcome/pull/37/commits/9a070ce154bb58140861e8623e31e5870cae9225